.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Let us know if something is broken on PyPI
+
+---
+
+<!--
+    NOTE: This issue should be for problems with PyPI itself, including:
+    * pypi.org
+    * test.pypi.org
+    * files.pythonhosted.org
+
+    This issue should NOT be for a project hosted on PyPI.  If you are having
+    an issue with a specific package, you should reach out to the maintainers
+    of that project directly instead.
```

Comment:
Sometimes "project hosted on PyPI" may not mean much. I'm trying to say something like "installed from PyPI using a tool like pip" (especially since 'pip install XXX' is so prominent on the project pages now).

Sometimes people are unsure how to find the maintainers. Perhaps one sentence here about looking for contact information on the PyPI project page? Now that we have multiple supported link types in the GUI, suggesting looking for the 'Bug Tracker' or 'Homepage' link might be good (hopefully lots of projects will adopt 'Bug Tracker' --- gevent did). 

Adding the above may start to get into wall-of-text-eyes-glaze-over territory, though, so I'm also happy with the present wording.

(Of course I'd like to see that 'NOT' *really* emphasised 😄 )

These LGTM!